dags/tests/integration/test_discord_find_guild_id.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime, timedelta
+from unittest import TestCase
+
+from bson import ObjectId
+
+from hivemind_etl_helpers.src.db.discord.find_guild_id import (
+    find_guild_id_by_community_id,
+)
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestFindDiscordGuildId(TestCase):
+    def add_platform(self):
+        client = MongoSingleton.get_instance().get_client()
+
+        action = {
```

Comment:
Not sure how this relates to Hivemind. Maybe this is just a copied test from discord_analyzer?

Reply:
The same function was used in discord-analyzer too. As we're working with Community Centeric db structure I needed to add the function `find_guild_id_by_community_id` and its tests too.
The function is used across discord_vectorstore and discord_summary_vectorstore since they needed the guildId from the communityId